examples/server_kube/README.md

```diff
@@ -1,3 +1,10 @@
 # Kubernetes ConfigMap example
 
-TODO(michal): Document this.
\ No newline at end of file
+- First download [etcd](https://github.com/coreos/etcd/releases), extract and make it available on your `$PATH`. (might need to move to `/user/local/bin`)
```

Comment:
the server_kube example directory is actually about (file/volume based) configmap updates, etcd isn't needed
(kubernetes is)